src/PerformanceTests/Common.Data/IProfileExtensionMethods.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace Common
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Configuration;
+    using System.Text;
+
+    public static class ProfileExtensionMethods
+    {
+        public static string GetConnectionString(this IProfile currentProfile, string connectionStringName)
+        {
+            var environmentVariableConnectionString = Environment.GetEnvironmentVariable(connectionStringName);
+            if (!string.IsNullOrWhiteSpace((environmentVariableConnectionString)))
+            {
+                return environmentVariableConnectionString;
+            }
+
+            var applicationConfigConnectionString = ConfigurationManager.ConnectionStrings[connectionStringName].ConnectionString;
+            if (!string.IsNullOrWhiteSpace((applicationConfigConnectionString)))
+            {
+                return applicationConfigConnectionString;
+            }
+
+            return string.Empty;
```

Comment:
Should this throw, perhaps? If no connection string is defined, can we continue without?